src/memary/agent/base_agent.py

```diff
@@ -134,6 +131,16 @@ def __init__(
 
     def __str__(self):
         return f"Agent {self.name}"
+    
+    def load_llm_model(self, llm_model_name):
+        if llm_model_name == "gpt-3.5-turbo":
+            os.environ["OPENAI_API_KEY"] = os.getenv("OPENAI_API_KEY")
+            self.openai_api_key = os.environ["OPENAI_API_KEY"]
+            self.model_endpoint = "https://api.openai.com/v1"
+            self.llm = OpenAI(model="gpt-3.5-turbo-instruct")
+        else: # default to llama3
+            self.llm = Ollama(model="llama3", request_timeout=60.0) 
```

Comment:
To be clear, I think it is better to write the exact llm_model_name with `elif llm_model_name == "llama3"`.
And add `else: raise(You should provide proper llm_model_name)`.